app/src/app.tsx

```diff
@@ -166,6 +173,9 @@ export const App = (): React.ReactElement => {
               <Text variant='header3'>{getDateString()}</Text>
               <IconButton icon={<KibaIcon iconId='ion-chevron-forward' />} onClicked={onForwardClicked} isEnabled={startDate < defaultDate} />
             </Stack>
+            <Spacing variant={PaddingSize.Wide2} />
+            <Text variant='header3'>{`${convertedTotalTransactions} transactions in total`}</Text>
```

Comment:
I would suggest just inlining this 
```suggestion
            <Text variant='header3'>{`${numberWithCommas(totalTransactions)} transactions in total`}</Text>
```